GNUmakefile

```diff
@@ -54,6 +54,7 @@ clean:
 
 build:
 	mkdir -p ${DIST_DIR}
+	go mod tidy
```

Comment:
Added this to build part because if our go mod gets out of date I want to make sure go mod tidy runs